src/multipleCenters.js

```diff
@@ -0,0 +1,68 @@
+// array of all centers
+const centers = []
+
+// here is what the object of the individual place looks like
+function center(sport) {
+  this.id = centers.length
+  this.sport = sport
+  this.enteringCheck = (adultsEntering, childrenEntering) => adultsEntering >= childrenEntering ? true : false
+  this.enter = (adultsEntering, childrenEntering) => {
+    if (this.enteringCheck(adultsEntering, childrenEntering)) {
+      this.currentPresence.adults += adultsEntering
+      this.currentPresence.children += childrenEntering
+      this.totalTally.adults += adultsEntering
+      this.totalTally.children += childrenEntering
+      return true
+    } else {
+      return false
+    }
+  }
+  this.leavingCheck = (adultsLeaving, childrenLeaving) => (adultsLeaving >= childrenLeaving && this.currentPresence.adults - childrenLeaving >= this.currentPresence.children - childrenLeaving) ? true : false
+  this.leave = (adultsLeaving, childrenLeaving) => {
+    if (this.leavingCheck(adultsLeaving, childrenLeaving)) {
+      this.currentPresence.adults -= adultsLeaving
+      this.currentPresence.children -= childrenLeaving
+      return true
+    } else {
+      return false
+    }
+  }
+  this.currentPresence = {
+    adults: 0,
+    children: 0
+  }
+  this.totalTally = {
+    adults: 0,
+    children: 0
+  }
+}
+
+const addCenter = (sport) => centers.push(new center(sport))
```

Comment:
This usage of the `new` keyword works as expected and is fine to create a new `instance` of a center; later on, you might discover that JS has a way to represent instances of objects with the concept of `class`, which would be very appropriate in this situation. You are very welcome to explore that if you want to get a taste of object oriented programming :) Keep in mind that it's all syntactical sugar, which means that at the end of the day, a class is just a fancier way to represent an object/function